content/en/docs/reference/glossary/compute-resource.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Resource (compute)
+id: compute-resource
+date: 2025-02-09
+full_link: /docs/concepts/architecture/nodes/#node-capacity
+short_description: >
+  A capability of a node availble for consumption (CPU, memory, etc).
+
+aka:
+tags:
+- fundamental
+---
+Capability of {{< glossary_tooltip text="nodes" term_id="node" >}} available for consumption (CPU, memory, etc) by processes or users of those nodes.
```

Comment:
Because of [dynamic resource allocation](https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) these capabilities may not be node local.

For example, you can allocate an attachable GPU as a resource, even though 8 different nodes are all able to request attachment.